libraries/Microsoft.Bot.Builder.Dialogs.Declarative/IComponentDeclarativeTypes.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.Bot.Builder.Dialogs.Declarative.Resources;
+using Newtonsoft.Json;
+
+namespace Microsoft.Bot.Builder.Dialogs.Declarative
+{
+    /// <summary>
+    /// Interface for registering declarative kinds and jsonconverters to support them.
+    /// </summary>
+    public interface IComponentDeclarativeTypes 
```

Comment:
>IComponentDeclarativeTypes [](start = 21, length = 26)

Class or interface?

Reply:
it's an interface.

---
In reply to: [376563073](https://github.com/microsoft/botbuilder-dotnet/pull/3350#discussion_r376563073) [](ancestors = 376563073)